csi/src/server.rs

```diff
@@ -109,8 +111,19 @@ impl AsyncWrite for UnixStream {
     }
 }
 
-#[tokio::main]
-async fn main() -> Result<(), String> {
+const GRPC_PORT: u16 = 10199;
+/// If endpoint is missing a port number then add the default one.
+pub fn grpc_endpoint(endpoint: String) -> std::net::SocketAddr {
+    (if endpoint.contains(':') {
```

Comment:
it is not common to use anonymous functions this way (i did not even know it was possible to be honest) what about something like this:
```
let s_addr = if endpoint.containers(':') { endpoint } else { format!( ... ) };
s_addr.parse().expect("...")
```

Reply:
Done

Reply:
Done